osquery/config/config_tests.cpp

```diff
@@ -22,12 +22,15 @@ namespace osquery {
 
 // The config_path flag is defined in the filesystem config plugin.
 DECLARE_string(config_path);
+// The config_extra_files flag is defined in the filesystem config plugin.
+DECLARE_string(config_extra_files);
```

Comment:
Actually, I think we are good to go with just using config_path + ".d/", thoughts?